src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static Task[] tasks = new Task[100];
+    public static int numTasks;
+
+    public static void printMessage(String message) {
+        String s = "     ";
+        String line = "    _____________________________________"
+                + "__________________________________\n";
+        System.out.println(line);
+        message = message.replace("\n", "\n" + s);
+        System.out.println(s + message);
+        System.out.println(line);
+    }
+
+    public static void printWelcomeMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String welcomeMessage = "Hello from \n" + logo + "\n"
+                + "What can I do for you?";
+        printMessage(welcomeMessage);
+    }
+
+    public static void printGoodbyeMessage() {
+        printMessage("Bye. Hope to see you again soon!");
+    }
+
+    public static void listTasks() {
+        int i = 0;
+        String listTasksMessage = "Here are the tasks in your list: \n";
+        while (i < numTasks) {
+            Task t = tasks[i];
+            i++;
+            listTasksMessage += String.valueOf(i) + ". " + t.toString() + "\n";
+        }
+        printMessage(listTasksMessage);
+    }
+
+    public static void addTask(String userCommand, String userDescription) {
+        if (String.valueOf(userDescription).equals("") == true) {
+            printMessage("Oops, the description of a '" + userCommand + "' cannot be empty.");
+            return;
+        }
+        Task t;
+        String taskDescription;
+        SplitString parsedUserDescription;
+        
+        switch (userCommand) {
+            case "todo":
```

Comment:
As in the previous comment, the case statements should follow the same indentation as the switch.